CHANGELOG.md

```diff
@@ -14,3 +14,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Build barcode scanner
 - Track patient vaccine status
 - Display immunization info on the Patient Profile 
+- Display message on empty list along with new client register button
```

Comment:
I think this should go under **Unreleased** for now instead of under **[0.0.3] - 2021-05-24** and when we release a new build, we can then move it under the release version.